package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "polymer-decorators",
+  "version": "0.1.0",
+  "description": "TypeScript decorators for Polymer 2.0",
+  "main": "lib/decorators.js",
+  "types": "lib/decorators.d.ts",
+  "scripts": {
+    "clean": "rm -rf lib global.js polymer-decorators*.tgz test/integration/node_modules",
+    "format": "find src -name '*.ts' | xargs clang-format --style=file -i",
+    "build": "npm run clean && tsc && rollup -c",
+    "prepack": "npm run build",
+    "test": "npm pack && mv polymer-decorators{-*,}.tgz && cd test/integration && npm install --no-package-lock && npm test"
```

Comment:
I thought `mv` had two arguments?

Reply:
This syntax creates two arguments 😃 

https://www.gnu.org/software/bash/manual/html_node/Brace-Expansion.html

Reply:
![](https://media.giphy.com/media/ujUdrdpX7Ok5W/giphy.gif)

Reply:
Switched to explicit syntax.

Reply:
But used it again for a more intuitive case 😄  (the clean command).